.circleci/config.yml

```diff
@@ -760,22 +752,6 @@ jobs:
       - DEPLOY_DOWNTIME: 1800 #30M
     steps: *deploy_steps
 
-  deploy_next:
-    executor: infrastructure_container_stable
-    <<: *infrastructure_config
-    environment:
-      - DEPLOY_ENV: next
-      - DEPLOY_DOWNTIME: 900 #15M
-    steps: *deploy_steps
-
-  deploy_unstable:
```

Comment:
This could be kept if amended config.
See proposal at https://www.pivotaltracker.com/story/show/164861501/comments/200893373

Reply:
I prefer to update the config first in scope of the other ticket and then re-add this.